ddp-workspace/projects/ddp-testboston/src/app/app.module.ts

```diff
@@ -85,6 +85,7 @@ sdkConfig.doLocalRegistration = DDP_ENV.doLocalRegistration;
 sdkConfig.mapsApiKey = DDP_ENV.mapsApiKey;
 sdkConfig.auth0Audience = DDP_ENV.auth0Audience;
 sdkConfig.projectGAToken = DDP_ENV.projectGAToken;
+sdkConfig.country = 'US';
 
```

Comment:
testboston limited to US

Reply:
May we use a bit more detailed name? For example, supportedRegion/supportedCountry

Reply:
6733cb5